### ISSUE TITLE
I love all the colours except the yellow - it is hard to look at - is there anything we can do to it?

### ISSUE THREAD

**Issue by joethorley** [poissonconsulting/poispalette#62]
<img width="544" alt="Screenshot 2023-01-20 at 09 44 27" src="https://user-images.githubusercontent.com/613671/213769774-a092693c-c9e6-48cf-b2fc-88cd078f995c.png">

**Comment by evanamiesgalonski** [comment]
Yes I think i agree looking at this. When I have some time I'll see if i can darken or perhaps change the hue a little without compromising our contrast for the colour blind accommodation

**Comment by evanamiesgalonski** [comment]
note: in the interim, the functions can take a vector of hex colours as palettes if you want to tweak it

**Comment by StefanoMezzini** [comment]
Do we want to close this since the PR has been approved?

**Comment by joethorley** [comment]
Yes - done!
